ydb/src/auth_test.rs

```diff
@@ -0,0 +1,9 @@
+use tracing_test::traced_test;
+
+use crate::YdbResult;
+
+#[tokio::test]
+#[traced_test]
+async fn auth_test() -> YdbResult<()> {
+    Ok(())
```

Comment:
Need real tests:
1. "SELECT 1" with good static_credentials - must be ok
2. "SELECT 1" with bad static_credentials (use bad password for user) - must failed